apps/src/code-studio/components/SoundLibrary.jsx

```diff
@@ -7,14 +7,41 @@ import Sounds from '../../Sounds';
 import firehoseClient from '@cdo/apps/lib/util/firehose';
 
 const SOUND_CATEGORIES = {
+  category_accent: 'Accent',
+  category_achievements: 'Achievements',
+  category_alerts: 'Alerts',
   category_animals: 'Animals',
+  category_app: 'App',
   category_background: 'Background',
+  category_bell: 'Bell',
   category_board_games: 'Board games',
+  category_collect: 'Collect',
   category_digital: 'Digital',
+  category_explosion: 'Explosion',
   category_female_voiceover: 'Female voiceovers',
+  category_hits: 'Hits',
+  category_human: 'Human',
   category_instrumental: 'Instrumental',
+  category_jump: 'Jump',
+  category_loops: 'Loops',
   category_male_voiceover: 'Male voiceovers',
+  category_movement: 'Movement',
+  category_music: 'Music',
+  category_nature: 'Nature',
+  category_notifications: 'Notifications',
   category_objects: 'Objects',
+  category_points: 'Points',
+  category_poof: 'Poof',
+  category_pop: 'Pop',
+  category_projectile: 'Projectile',
+  category_puzzle: 'Puzzle',
+  category_retro: 'Retro',
+  category_slide: 'Slide',
+  category_swing: 'Swing',
+  category_swish: 'Swish',
+  category_tap: 'Tap',
+  category_transition: 'Transition',
+  category_whoosh: 'Whoosh',
   category_all: 'All'
```

Comment:
I would not address in _this_ PR, but these category names look like something that should be translated.  I'd chat with @joshlory about potential followup work here localizing these strings.

Reply:
I will check in with Josh about this! Thanks for letting me know!

Reply:
Based on discussion with Josh, localization will be something that volunteers could potentially work on. For this PR, the category names will be kept the same but later it will be translated.